packages/overlays/test/OverlayController.test.js

```diff
@@ -430,6 +430,8 @@ describe('OverlayController', () => {
         // Don't hide on inside (content) click
         ctrl.contentNode.click();
         await aTimeout();
```

Comment:
These two timeouts are necessary for the double setTimeout in the src? So if we removed these timeouts here in the test, and revert the fix, this test would pass, there we keep the timeouts in this test so it's a proper regression test? :) Just for my understanding

Reply:
They are not neccessary (anymore), removed them 👍